src/main/kotlin/org/jetbrains/kotlinx/jupyter/repl.kt

```diff
@@ -205,6 +208,10 @@ class ReplForJupyterImpl(
 
     private val resourcesProcessor = LibraryResourcesProcessorImpl()
 
+    init {
+        registerKnownCommsHandlersOnStartUp(this)
```

Comment:
I'd like to refactor repl class in a way it would be fully configurable via factory. Please make it possible to configure (switch off/on, add specific handlers) registration of these comms via factory